ui/app/styles/core/title.scss

```diff
@@ -11,6 +11,10 @@
     color: $black;
     text-decoration: none;
   }
+
+  &.title-has-bottom-margin-xs {
```

Comment:
Could you instead add the `has-bottom-margin-xs` class to the title? Looks like it has the same style definition

Reply:
Good catch! Thank you.